File: backend/app/schemas/helm.py
from pydantic import BaseModel


class HelmRelease(BaseModel):
    name: str
    namespace: str
    chart: str
    chart_version: str | None
    app_version: str | None
    status: str
    revision: int
    description: str | None
    first_deployed: str | None
    last_deployed: str | None

File: frontend/src/types/helm.ts
export interface HelmRelease {
  name: string;
  namespace: string;
  chart: string;
  chart_version: string | null;
  app_version: string | null;
  status: string;
  revision: number;
  description: string | null;
  first_deployed: string | null;
  last_deployed: string | null;
}
